src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -121,6 +218,14 @@ private static string GetClassificationForIdentifier(SyntaxToken token)
             {
                 return IsExtensionMethod(methodDeclaration) ? ClassificationTypeNames.ExtensionMethodName : ClassificationTypeNames.MethodName;
             }
+            else if (token.Parent is ConstructorDeclarationSyntax constructorDeclaration && constructorDeclaration.Identifier == token)
+            {
+                return ClassificationTypeNames.MethodName;
```

Comment:
this doesn't seem right.   Perhaps we should have 'ContructorName'?  Same comment about below.